relays/lib-substrate-relay/src/finality_base/engine.rs

```diff
@@ -110,6 +124,13 @@ pub trait Engine<C: Chain>: Send {
 		header: &C::Header,
 		proof: &mut Self::FinalityProof,
 	) -> Result<(), SubstrateError>;
+	/// Returns `true` if selected header and proof are exceeding maximal expected
```

Comment:
Nit: I would add a new line after the previous method declaration.

Reply:
Nit: also "Returns `true`` vs `MaxExpectedCallSizeCheck`  is not valid, right?